app/containers/message/Components/Attachments/ImageGallery/ImageGallery.tsx

```
@@ -0,0 +1,86 @@
+	getCustomEmoji?: TGetCustomEmoji;
+}
+
+const ImageGallery = ({ files, showAttachment, author, getCustomEmoji }: IImageGallery) => {
+	const [containerWidth, setContainerWidth] = useState(0);
+
+	if (containerWidth === 0) {
+		return (
+			<View
+				style={{ height: GALLERY_HEIGHT }}
+				onLayout={ev => setContainerWidth(Math.floor(ev.nativeEvent.layout.width))}
+			/>
```

ESLint

[error] Replace `⏎↹↹↹↹style={{·height:·GALLERY_HEIGHT·}}⏎↹↹↹↹onLayout={ev·=>·setContainerWidth(Math.floor(ev.nativeEvent.layout.width))}⏎↹↹↹` with `·style={{·height:·GALLERY_HEIGHT·}}·onLayout={ev·=>·setContainerWidth(Math.floor(ev.nativeEvent.layout.width))}·`

(prettier/prettier)